src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlUtil.cs

```diff
@@ -358,6 +359,12 @@ sealed internal class InOutOfProcHelper
         [ResourceConsumption(ResourceScope.Process, ResourceScope.Process)]
         private InOutOfProcHelper()
         {
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
the [RuntimeInformation.IsOSPlatform](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.runtimeinformation.isosplatform?view=net-5.0) is supported in .Net Framework starting from 4.7.1 and our driver is built on 4.6.1.

Reply:
Indeed, good catch. Would it be OK to take a dependency on [System.Runtime.InteropServices.RuntimeInformation 4.3.0](https://www.nuget.org/packages/System.Runtime.InteropServices.RuntimeInformation/4.3.0) which has an implementation of `RuntimeInformation.IsOSPlatform` for `net45` (and thus supports .NET Framework 4.6.1) ?

Reply:
We have used `System.Runtime.InteropServices.RuntimeInformation` in several places in our driver, like in `SqlBuffer `or `SqlConnection` itself in `netfx` section. Change your code accordingly and there should not be any problem I think. With more detail, if you look inside the `Version.props `folder we have the dependency `<SystemRuntimeInteropServicesRuntimeInformationVersion>4.3.0</SystemRuntimeInteropServicesRuntimeInformationVersion>`

Reply:
Probably that's the reason why pipelines did not complain :)